apps/nocobase/1.4.10/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   nocobase:
-    image: nocobase/nocobase:1.4.10
+    image: nocobase/nocobase:1.4.11
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The changes between the two lines are that the version of `nocobase/nocobase` has been updated from `1.4.10` to `1.4.11`. This change is minor and does not introduce any potential issues. If you need further optimizations or advice based on this specific update, let me know!